src/main/java/com/google/flourbot/api/DriveCloudSheet.java

```diff
@@ -0,0 +1,60 @@
+package com.google.flourbot.api;
+
+import com.google.api.services.sheets.v4.*;
+import com.google.api.services.sheets.v4.model.*;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.logging.Logger;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+class DriveCloudSheet implements CloudSheet {
+  Sheets sheetsService;
+  String spreadsheetId;
+
+  public DriveCloudSheet(String documentId, Sheets documentService) {
+    this.spreadsheetId = documentId;
+    this.sheetsService = documentService;
+  }
+
+  // TODO: replace hardcoded range with calculated range (so far values must be length 3)
+  public void appendRow(List<String> values) throws IOException, GeneralSecurityException {
+    
+    // Generate request to append to sheet
+    String range = "Sheet1!A1:C1";
```

Comment:
not sure why the range is needed here when we can append lists of arbitrary length